### PR TITLE
fix nightly build of router crate

### PR DESCRIPTION
**Pull request by miller-time**
closes https://github.com/iron/router/issues/148

- update route-recognizer (implicitly)
- add `dyn` to trait objects

### CODE REVIEW

File: router/Cargo.toml [hunk 1]
```diff
@@ -10,6 +9,6 @@ documentation = "http://ironframework.io/doc/router/index.html"
 keywords = ["iron", "web", "http", "routing", "router"]
 
 [dependencies]
-route-recognizer = "0.1"
+route-recognizer = "0.1.13"
```
Comment (phlmn) [inline router/Cargo.toml]: There should be no need for specifying a patch version, `cargo update` should be enough.

**Comment by miller-time** [comment]
@phlmn I think "router" needs to be added to:

https://github.com/iron/iron/blob/cf056ea5e8052c1feea6141e40ab0306715a2c33/Cargo.toml#L3

should I add that to this PR?

**Comment by phlmn** [comment]
@miller-time Yep, noticed that, I've just pushed the fix (https://github.com/iron/iron/commit/73fbcec1a5d57d9af0899147047a36ffd7a5cc16).

**Comment by miller-time** [comment]
rebased :nerd_face:

**Comment by phlmn** [comment]
beautiful ❤️

**Pull request merged**